### ISSUE TITLE
ImportError: numpy.core.multiarray failed to import

### ISSUE THREAD

**Issue by bernd-wechner** [wolfmanstout/screen-ocr#11]
On Windows 10:

```
(Testing) PS E:\> pip install screen-ocr[winrt]
Collecting screen-ocr[winrt]
  Downloading screen_ocr-0.3.0-py3-none-any.whl (15 kB)
Collecting rapidfuzz
  Downloading rapidfuzz-2.13.2-cp310-cp310-win_amd64.whl (1.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 4.1 MB/s eta 0:00:00
Requirement already satisfied: pillow in e:\venvs\testing\lib\site-packages (from screen-ocr[winrt]) (9.3.0)
Collecting screen-ocr[winrt]
  Downloading screen_ocr-0.2.0-py2.py3-none-any.whl (13 kB)
Collecting scikit-image
  Downloading scikit_image-0.19.3-cp310-cp310-win_amd64.whl (12.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.0/12.0 MB 7.5 MB/s eta 0:00:00
Requirement already satisfied: numpy in e:\venvs\testing\lib\site-packages (from screen-ocr[winrt]) (1.21.4)
Collecting screen-ocr[winrt]
  Downloading screen_ocr-0.1.1-py2.py3-none-any.whl (12 kB)
Requirement already satisfied: pandas in e:\venvs\testing\lib\site-packages (from screen-ocr[winrt]) (1.3.4)
  Downloading screen_ocr-0.1.0-py2.py3-none-any.whl (12 kB)
WARNING: screen-ocr 0.1.0 does not provide the extra 'winrt'
Collecting pytesseract
  Downloading pytesseract-0.3.10-py3-none-any.whl (14 kB)
Requirement already satisfied: pytz>=2017.3 in e:\venvs\testing\lib\site-packages (from pandas->screen-ocr[winrt]) (2021.3)
Requirement already satisfied: python-dateutil>=2.7.3 in e:\venvs\testing\lib\site-packages (from pandas->screen-ocr[winrt]) (2.8.2)
Collecting packaging>=21.3
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting scipy>=1.4.1
  Downloading scipy-1.9.3-cp310-cp310-win_amd64.whl (40.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.1/40.1 MB 20.4 MB/s eta 0:00:00
Collecting PyWavelets>=1.1.1
  Downloading PyWavelets-1.4.1-cp310-cp310-win_amd64.whl (4.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.2/4.2 MB 29.5 MB/s eta 0:00:00
Collecting networkx>=2.2
  Downloading networkx-2.8.8-py3-none-any.whl (2.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.0/2.0 MB 25.5 MB/s eta 0:00:00
Collecting imageio>=2.4.1
  Downloading imageio-2.22.4-py3-none-any.whl (3.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.4/3.4 MB 30.8 MB/s eta 0:00:00
Collecting tifffile>=2019.7.26
  Downloading tifffile-2022.10.10-py3-none-any.whl (210 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 210.3/210.3 kB 6.5 MB/s eta 0:00:00
Collecting pyparsing!=3.0.5,>=2.0.2
  Using cached pyparsing-3.0.9-py3-none-any.whl (98 kB)
Requirement already satisfied: six>=1.5 in e:\venvs\testing\lib\site-packages (from python-dateutil>=2.7.3->pandas->screen-ocr[winrt]) (1.16.0)
Installing collected packages: tifffile, scipy, rapidfuzz, PyWavelets, pyparsing, networkx, imageio, packaging, scikit-image, pytesseract, screen-ocr
Successfully installed PyWavelets-1.4.1 imageio-2.22.4 networkx-2.8.8 packaging-21.3 pyparsing-3.0.9 pytesseract-0.3.10 rapidfuzz-2.13.2 scikit-image-0.19.3 scipy-1.9.3 screen-ocr-0.1.0 tifffile-2022.10.10
```

Quick test spin and immediate failure:

```
>>> import screen_ocr
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "E:\venvs\Testing\lib\site-packages\screen_ocr\__init__.py", line 1, in <module>
    from ._screen_ocr import *
  File "E:\venvs\Testing\lib\site-packages\screen_ocr\_screen_ocr.py", line 7, in <module>
    from skimage import filters, morphology, transform
  File "E:\venvs\Testing\lib\site-packages\skimage\__init__.py", line 157, in <module>
    from .util.dtype import (img_as_float32,
  File "E:\venvs\Testing\lib\site-packages\skimage\util\__init__.py", line 16, in <module>
    from ._montage import montage
  File "E:\venvs\Testing\lib\site-packages\skimage\util\_montage.py", line 4, in <module>
    from .. import exposure
  File "E:\venvs\Testing\lib\site-packages\skimage\_shared\lazy.py", line 62, in __getattr__
    return importlib.import_module(f'{package_name}.{name}')
  File "E:\Python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "E:\venvs\Testing\lib\site-packages\skimage\exposure\__init__.py", line 6, in <module>
    from ._adapthist import equalize_adapthist
  File "E:\venvs\Testing\lib\site-packages\skimage\exposure\_adapthist.py", line 21, in <module>
    from ..color.adapt_rgb import adapt_rgb, hsv_value
  File "E:\venvs\Testing\lib\site-packages\skimage\color\__init__.py", line 1, in <module>
    from .colorconv import (convert_colorspace,
  File "E:\venvs\Testing\lib\site-packages\skimage\color\colorconv.py", line 55, in <module>
    from scipy import linalg
  File "E:\venvs\Testing\lib\site-packages\scipy\__init__.py", line 225, in __getattr__
    return _importlib.import_module(f'scipy.{name}')
  File "E:\Python\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "E:\venvs\Testing\lib\site-packages\scipy\linalg\__init__.py", line 197, in <module>
    from ._misc import *
  File "E:\venvs\Testing\lib\site-packages\scipy\linalg\_misc.py", line 3, in <module>
    from .blas import get_blas_funcs
  File "E:\venvs\Testing\lib\site-packages\scipy\linalg\blas.py", line 213, in <module>
    from scipy.linalg import _fblas
ImportError: numpy.core.multiarray failed to import
```

For what it's worth if it helps diagnose:

```
(Testing) PS E:\> python --version
Python 3.10.4
(Testing) PS E:\> pip freeze
beautifulsoup4==4.11.1
cftime==1.5.1.1
chardet==5.0.0
click==8.1.3
colorama==0.4.5
comtypes==1.1.11
decorator==5.1.1
elevate==0.1.3
func-timeout==4.3.5
humanfriendly==10.0
humanize==4.1.0
imageio==2.22.4
lxml==4.9.1
netCDF4==1.5.8
networkx==2.8.8
nose==1.3.7
numpy==1.21.4
packaging==21.3
pandas==1.3.4
pathvalidate==2.5.0
Pillow==9.3.0
psutil==5.9.4
pynmea2==1.18.0
pyodbc==4.0.32
pyparsing==3.0.9
pypyodbc==1.3.6
pyreadline3==3.4.1
pytesseract==0.3.10
python-dateutil==2.8.2
python-magic-bin==0.4.14
pytz==2021.3
pyuac==0.0.3
PyWavelets==1.4.1
pywin32==304
pywinauto==0.6.8
rapidfuzz==2.13.2
scikit-image==0.19.3
scipy==1.9.3
screen-ocr==0.1.0
six==1.16.0
soupsieve==2.3.2.post1
svn==1.0.1
tee==0.0.3
tifffile==2022.10.10
winregistry==1.1.1
```

**Comment by bernd-wechner** [comment]
For what it's worth this fixed it:

```
pip install numpy --upgrade
```

But if the screen-ocr had the right numpy dependency I'm guessing that would have never arisen as an issue and been dealt with when I installed screen-ocr.

**Comment by wolfmanstout** [comment]
The main issue here is that winrt has not yet been released for Python 3.10, so pip is falling back to an old version of my package. There's no easy fix for this from my end; see this for details: https://github.com/wolfmanstout/screen-ocr/issues/10

For posterity: the error you're seeing is really a bug in scikit-image where it's not declaring its numpy version requirements (while I do depend on numpy directly, it's not my import that's causing the problem here). The latest version of my package doesn't depend on scikit-image by default (only if you use the tesseract backend), so this issue will go away if you use a compatible version of Python (e.g. 3.9). I'll add a note to my readme so that other folks don't get tripped up on this.
